tools/start-stop-daemon/start-stop-daemon.c

```diff
@@ -39,6 +39,8 @@
 #  define OSFreeBSD
 #elif defined(__NetBSD__)
 #  define OSNetBSD
+#elif defined(__APPLE__)
+#  define OSLinux
```

Comment:
I thought the same thing.  When using OSFreeBSD, I got the following incompatibilities:

In file included from /Users/bone/git/boneill42/hpcc/tools/start-stop-daemon/start-stop-daemon.c:65:
/usr/include/sys/proc.h:118:19: error: field has incomplete type 'struct itimerval'
        struct  itimerval p_realtimer;  /* Alarm timer. */
                          ^
/usr/include/sys/proc.h:118:9: note: forward declaration of 'struct itimerval'
        struct  itimerval p_realtimer;  /* Alarm timer. */
                ^
/Users/bone/git/boneill42/hpcc/tools/start-stop-daemon/start-stop-daemon.c:1263:9: warning: implicit declaration of function 'do_procinit' is invalid in C99 [-Wimplicit-function-declaration]
        do_procinit();